cdc/streamer.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
+
+import (
+	"time"
+
+	"github.com/apple/foundationdb/bindings/go/src/fdb"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/rs/zerolog/log"
+	"github.com/tigrisdata/tigrisdb/internal"
+	"github.com/tigrisdata/tigrisdb/server/config"
+)
+
+type Streamer struct {
+	db      fdb.Database
```

Comment:
will you be changing it to use kvstore in the next iteration?

Reply:
yeah, need to extend kvstore to support necessary functions